.husky/pre-commit

```diff
@@ -6,3 +6,11 @@ npx lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependen
 
 # For root directory, which is not managed by lerna
 npx lint-staged
+
+# For subdirectory, only if there are changes
```

Comment:
The `npx lerna run ... precommit` above is running lint-staged in the subdirectories -- if you check out the run scripts you will see that this is what is defined in `precommit`.

I would recommend following the same pattern as the other packages (adding that `precommit` script to `package.json` -- definitely over conditional bash scripting!) because otherwise we are defining two ways to do the same thing in this file.

Is there a reason the pre-existing pattern doesn't work here?